samples/tfm_integration/README.rst

```diff
@@ -0,0 +1,88 @@
+.. _tfm_integration:
+
+TF-M Integration
+################
+
+Overview
+********
+This is a simple TF-M integration example that can be used with an Arm v8-M
+supported board.
+Zephyr uses Trusted Firmware (TF-M) Platform Security Architecture (PSA) APIs
+to run this sample in a secure configuration with Zephyr itself in a
+non-secure configuration.
+The sample prints test info to the console either as a single- or
+multi-threaded application.
+
+Building and Running
+********************
+
+This project outputs test status and info to the console. It can be built and
+executed on MPS2+ AN521, Musca B1.
```

Comment:
TODO: Better to add an item to list specific command lines about how to build TFM per the suggestion from @microbuilder, to make it clearer for anyone to run this sample easily.